scalafmt-cli/src/main/scala/org/scalafmt/cli/CliArgParser.scala

```diff
@@ -35,40 +35,55 @@ object CliArgParser {
     new scopt.OptionParser[CliOptions]("scalafmt") {
       override def showUsageOnError = false
 
-      def printAndExit(inludeUsage: Boolean)(ignore: Unit,
-                                             c: CliOptions): CliOptions = {
+      private def printAndExit(
+          inludeUsage: Boolean)(ignore: Unit, c: CliOptions): CliOptions = {
         if (inludeUsage) showUsage
         else showHeader
         sys.exit
         c
       }
 
-      def readConfigFromFile(file: String, c: CliOptions): CliOptions = {
+      private def readConfigFromFile(file: String, c: CliOptions): CliOptions = {
         readConfig(
           FileOps.readFile(
             AbsoluteFile.fromFile(new File(file), c.common.workingDirectory)),
           c
         )
       }
-      def readConfig(contents: String, c: CliOptions): CliOptions = {
+      private def readConfig(contents: String, c: CliOptions): CliOptions = {
         c.copy(config = Config.fromHoconString(contents).get)
       }
 
+      private def addFile(file: File, c: CliOptions): CliOptions = {
+        val absFile = AbsoluteFile.fromFile(file, c.common.workingDirectory)
+        c.copy(customFiles = c.customFiles :+ absFile)
+      }
+
       head("scalafmt", Versions.nightly)
       opt[Unit]('h', "help")
         .action(printAndExit(inludeUsage = true))
         .text("prints this usage text")
       opt[Unit]('v', "version")
         .action(printAndExit(inludeUsage = false))
         .text("print version ")
-      opt[Seq[File]]('f', "files")
-        .action { (files, c) =>
-          c.copy(
-            customFiles =
-              AbsoluteFile.fromFiles(files, c.common.workingDirectory))
-        }
+
+      arg[File]("<file>...")
+        .optional()
+        .unbounded()
+        .action((file, c) => addFile(file, c))
         .text(
           "file or directory, in which case all *.scala files are formatted.")
+
+      opt[Unit]('i', "in-place")
```

Comment:
Ohh, I see now what you mean. Yeah, it doesn't make sense to have it anymore since it is the default. 

However, if we are keeping `-f` then we should keep `-i` (since a lot of current scripts probably have `-i -f`).